articles/understanding-static-dynamic-routing/index.md

```diff
@@ -0,0 +1,276 @@
+# Understanding Dynamic and Static Routing
+
+![Hero image](/engineering-education/understanding-dynamic-and-static-routing/hero.jpeg)
+
+[Source](https://unsplash.com/photos/IL1qSqEMNBo)
+
```

Comment:
@raboba2re, remove the 'hero' image., should not be included in this file.(Note: DO NOT delete it from your article folder).